conda.recipe/meta.yaml

```diff
@@ -67,6 +67,7 @@ test:
     - selenium >=3
     - pytest-xdist
     - pytest-rerunfailures
+    - requests >=1.2.3
```

Comment:
Turns out requests is part of miniconda / required by conda. So that's why it works for our tests on travis with this removed. I worry that someone working in a non-root conda env, or without conda at all could get tripped up by this being missing. I'd vote to leave this in as a test-dep only.